pages/common/git-log.md

```diff
@@ -13,3 +13,7 @@
 - Show only the first line of each commits:
 
 `git log --oneline`
+
+- Show branch graph:
```

Comment:
How about "Show the graph of commits in the current branch"  ?

Reply:
Thanks for your suggestion.